modules/ocf/manifests/node_exporter.pp

```diff
@@ -7,10 +7,8 @@
     content => template('ocf/environment.prom.erb'),
   }
 
+  # Attributes for this class are defined in hieradata/common.yaml
```

Comment:
I'd move this right before the class (inside the conditional), just so it stays closer to the class that it's referring to.

nit: Just saying "in hieradata" is probably better as it could be defined at any level of hiera, not just in `common.yaml` (though that's currently the only place it's defined).